pangocairo/Gir.toml

```diff
@@ -106,15 +106,21 @@ name = "PangoCairo.FontMap"
 status = "generate"
 manual_traits = ["FontMapExtManual"]
     [[object.function]]
+    # Needed because cairo types don't implement `.into_glib()`.
```

Comment:
I am a bit late but shouldn't it be possible to implement such traits for cairo types somehow?